Dockerfile-custom

```diff
@@ -0,0 +1,9 @@
+FROM localhost/maestro-base:latest
+
+# COPY <src> agent.yaml      # copy agent file into image
+# COPY <dest> workflow.yaml  # copy workflow file into image
+
+CMD ["serve", "agent.yaml", "workflow.yaml", "--port=5000"]  # for workflow serve
+#CMD ["serve", "agent.yaml", "--port=5000"]                  # for agent serve
+
+ENTRYPOINT ["maestro"]
```

Comment:
IIUC this would run `ENTRYPOINT CMD` when docker run?

So this could work out of the box if placed in a dir with a agent.yaml and workflow.yaml

It would also allow direct use by passing in different args to override CDM correct?

Reply:
Also if we uncomment the ENTRYPOINT in the absence of image would we even need this line?